submissions/OwoeyeGbenga/landing_page/css/style.css

```diff
@@ -0,0 +1,389 @@
+html {
+  box-sizing: border-box;
+}
+*, *:before, *:after {
+  box-sizing: inherit;
```

Comment:
The _box-sizing_ rule here overrides the one for _html_ tag above. You might want to remove that one

Reply:
Done